src/techs/techs.service.ts

```diff
@@ -163,53 +189,245 @@ export class TechsService {
         }
     }
 
-    async addExistingTechVote(req, teamId, teamTechId) {
+    async updateExistingTeamTech(
+        req: CustomRequest,
+        updateTeamTechDto: UpdateTeamTechDto,
+        teamTechItemId: number,
+    ) {
+        const { techName } = updateTeamTechDto;
         // check if team tech item exists
         const teamTechItem = await this.prisma.teamTechStackItem.findUnique({
             where: {
-                id: teamTechId,
+                id: teamTechItemId,
+            },
+            select: {
+                addedBy: {
```

Comment:
Also here I am getting an error saying "Object literal may only specify known properties, and 'addedBy' does not exist in type 'TeamTechStackItemSelect<DefaultArgs>'.ts(2353)"

Reply:
I think you have to run ` yarn migrate`  to apply the new migrations to dB .

Reply:
oh yes, that fixed them.